src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2001,9 +2005,13 @@ public String getResolution() {
 		return null;
 	}
 
-	public int getRealVideoBitrate() {
-		if (bitrate > 0) {
-			return (bitrate / 8);
+	// XXX This doesn't handle variable bitrates, nor does it adhere to DLNA which states that
+	// (for variable bitrates) the bitrate should be the maximum bitrate or if unknown, the maximum
```

Comment:
As a side note, i'll said that AAC variant have the `bitRate_Maximum`, that will be reworked in Mediainfo, and DTS VBR should be handled with the maximum value specified in the standard depending on its channels number.
That was just my point of view.